src/main/kotlin/blackjack/domain/score/Score.kt

```diff
@@ -0,0 +1,14 @@
+package blackjack.domain.score
+
+data class Score(
+    val value: Int
+) {
+
+    val isAlive = value <= WIN_NUMBER
+    val isBust = isAlive.not()
+
+    companion object {
+
+        private const val WIN_NUMBER = 21
```

Comment:
```suggestion
        private const val BLACKJACK = 21
```
블랙잭에서는 21이라는 숫자를 "블랙잭"이라고 합니다. 😄

Reply:
최초에 받은 2장의 카드가 21이 되었을 때를 블랙잭이라 표현하는 것으로 이해했는데요, 혹시 제가 잘못 이해한걸까요~?
https://namu.wiki/w/%EB%B8%94%EB%9E%99%EC%9E%AD(%EC%B9%B4%EB%93%9C%EA%B2%8C%EC%9E%84)#:~:text=%EC%B2%98%EC%9D%8C%202%EC%9E%A5%EC%9D%98%20%EC%B9%B4%EB%93%9C%EA%B0%80%20%EC%97%90%EC%9D%B4%EC%8A%A4%EC%99%80%2010(J%C2%B7Q%C2%B7K%EB%A5%BC%20%ED%8F%AC%ED%95%A8)%EC%9C%BC%EB%A1%9C%2021%EC%A0%90%EC%9D%B4%20%EB%90%9C%20%EA%B2%83%EC%9D%84%20%27%EB%B8%94%EB%9E%99%EC%9E%AD%27%EC%9D%B4%EB%9D%BC%EA%B3%A0%20%ED%95%98%EB%A9%B0%2C

Reply:
앗!
확인해보니 이 부분은 제가 잘못 인지하고 있었습니다. 🙏